packages/nns/candid/genesis_token.did

```diff
@@ -1,4 +1,4 @@
-// Generated from IC repo commit 987e1c810a9f6770fd1b2f1ae4a895d751d5ef41 'rs/nns/gtc/canister/gtc.did' by import-candid
+// Generated from IC repo commit a1080f27696c890c19d7068f668bd062f3787564 'rs/nns/gtc/canister/gtc.did' by import-candid
```

Comment:
No update for genesis, governance and ledger but these are auto-generated with the new scripts, so I commited the hash change anyway since the script updated it.